include/nbl/video/utilities/IGPUObjectFromAssetConverter.h

```diff
@@ -1089,16 +1022,34 @@ auto IGPUObjectFromAssetConverter::create(const asset::ICPUImage** const _begin,
         IGPUImage::SCreationParams params = {};
         params = cpuimg->getCreationParameters();
         params.initialLayout = asset::IImage::EL_UNDEFINED;
+        
+        IPhysicalDevice::SImageFormatPromotionRequest promotionRequest = {};
+        promotionRequest.originalFormat = params.format;
+        promotionRequest.usages = {};
 
         const bool integerFmt = asset::isIntegerFormat(params.format);
         if (!integerFmt)
             params.mipLevels = 1u + static_cast<uint32_t>(std::log2(static_cast<float>(core::max<uint32_t>(core::max<uint32_t>(params.extent.width, params.extent.height), params.extent.depth))));
 
-        if (cpuimg->getRegions().size() && !(params.usage.value & asset::IImage::EUF_TRANSFER_DST_BIT))
+        if (cpuimg->getRegions().size())
+        {
             params.usage |= asset::IImage::EUF_TRANSFER_DST_BIT;
+        }
 
         if (needToCompMipsForThisImg(cpuimg))
+        {
             params.usage |= asset::IImage::EUF_TRANSFER_SRC_BIT;
+            // TODO: revise the next two lines
+            promotionRequest.usages.blitDst = true;
+            promotionRequest.usages.blitSrc = true;
```

Comment:
also revise the `params.usage` (needs to be texture filtered fetch and image store)